class_exercise_1.rb

```diff
@@ -16,5 +16,37 @@ def multiples
   
 end
 
-multiples = MultiplesOf.new(3, 5, 1000)
-puts multiples.multiples
+#--------------TEST-------------------
+
+
+require 'minitest/autorun'
+
+class MultiplesTest < Minitest::Test
+
+  def setup
+    @num =  MultiplesOf.new(3, 5, 1000)
+  end
+
+  def test_numero_is_instance_of_MultiplesOf
+    assert_instance_of(MultiplesOf, @num)
+  end
+
+  def test_numero_should_have_resultado
+    assert_equal(233168, @num.multiples)
+  end
+
+end
+
+#----------------------------------------------
+
+describe MultiplesOf do
+  before do
+    @num1 =  MultiplesOf.new(3, 5, 1000)
```

Comment:
Why this variables is named num1? it could be named num and when you want a variable with a number, you should do something like : @num_1